ocs_ci/deployment/provider_client/storage_client_deployment.py

```diff
@@ -210,6 +242,14 @@ def provider_and_native_client_installation(
                 storage_cluster_data = self.add_encryption_details_to_cluster_data(
                     storage_cluster_data
                 )
+                storage_cluster_data["spec"]["storageDeviceSets"][0][
+                    "replica"
+                ] = no_of_worker_nodes
+
+                if self.platform == constants.BAREMETAL_PLATFORM:
```

Comment:
please change to `if self.platform in constants.HCI_PROVIDER_CLIENT_PLATFORMS:`
We should use only the next config when deploying a new cluster. 
`ENV_DATA:
    platform: "hci_baremetal"`
Full deployment config:
https://docs.google.com/document/d/1RfDNQi4B3x4kv9PXx2lqGSD2V2UGideTCIXqLYJFg0Y/edit#bookmark=id.5lrwe0xopvrx

Reply:
Updated in latest commit.